deeplabcut/utils/make_labeled_video.py

```diff
@@ -1168,15 +1266,42 @@ def create_video_with_all_detections(
 
             pcutoff = cfg["pcutoff"]
             dotsize = cfg["dotsize"]
-            clip = vp(fname=video, sname=outputname, codec="mp4v")
+            clip = vp(fname=video, sname=str(outputname), codec="mp4v")
             ny, nx = clip.height(), clip.width()
 
+            bboxes_pcutoff = metadata.get("data", {}).get("pytorch-config", {}).get("detector", {}).get("model", {}).get("box_score_thresh", 0.6)
+            bboxes_color = (0, 0, 0)
+
             for n in trange(clip.nframes):
                 frame = clip.load_frame()
                 if frame is None:
                     continue
                 try:
                     ind = frames.index(n)
+
+                    # Draw bounding boxes of required and present
+                    if plot_bboxes and "bboxes" in data[frame_names[ind]]:
+                        bboxes = data[frame_names[ind]]["bboxes"]
+                        bbox_scores = data[frame_names[ind]]["bbox_scores"]
+                        n_bboxes = bboxes.shape[0]
+                        for i in range(n_bboxes):
+                            bbox = bboxes[i,:]
+                            x, y = bbox[0], bbox[1]
+                            x += x1
+                            y += y1
+                            w, h = bbox[2], bbox[3]
+                            confidence = bbox_scores[i]
+                            if confidence < bboxes_pcutoff:
+                                continue
+                            rect_coords = rectangle_perimeter(start=(y, x), extent=(h, w))
+
+                            set_color(
+                                frame,
+                                rect_coords,
+                                bboxes_color,
+                            )
+
+                    # Draw detected bodyparts
```

Comment:
We have this plotting of labels (and bboxes) on frames at least 4 times:
- In _deeplabcut/utils/make_labeled_video.py_:
    - in `create_video()`
    - in `create_video_slow()`
    - in `create_video_with_all_detections()`
- In _deeplabcut/utils/visualization.py_:
    - in `make_multianimal_labeled_image()`
    
Even if we need 1 version based on Pyplot and another based on skimage, can we reduce the number of these functions to 2? And ideally to one?